manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
@chrisarcand again, you missed the part where I said:

> ... (in master)...

Basically, we have two different ways _**IN MASTER**_ in these projects for determining the version of Rails to use:

https://github.com/ManageIQ/manageiq-ui-classic/blob/master/manageiq-ui-classic.gemspec#L22

v.s.

https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L79

This is silly, and confusing.  This should be consistent, or you will have to do this more often in the future, and curse and such.